src/openrct2/interface/Viewport.cpp

```diff
@@ -1167,6 +1162,17 @@ namespace OpenRCT2
         return { mapCoords->ToTileStart() };
     }
 
+    [[nodiscard]] bool Viewport::ContainsTile(const TileCoordsXY coords) const noexcept
+    {
+        const auto centreCoords = coords.ToCoordsXY() + CoordsXY(16, 16);
+        const auto screenPos = Translate3DTo2DWithZ(rotation, CoordsXYZ{ centreCoords, 0 });
+        const auto left = screenPos.x - 32;
```

Comment:
Is there any significance to these values of `16` and `32`? If so, I think a constant would be a good thing?